test/unit/org/apache/cassandra/service/accord/AccordCommandStoreFuzzTest.java

```
@@ -0,0 +1,457 @@
+            orderedExecutor = globalExecutor.configureSequential("ignore").build();
+            unorderedScheduled = globalExecutor.scheduled("ignored");
+            ExecutorFactory.Global.unsafeSet(globalExecutor);
+            Stage.READ.unsafeSetExecutor(unorderedScheduled);
+            Stage.MUTATION.unsafeSetExecutor(unorderedScheduled);
+
+            this.updateHolder = new CommandStore.EpochUpdateHolder();
+            this.nodeId = AccordTopology.tcmIdToAccord(ClusterMetadata.currentNullable().myNodeId());
+            this.timeService = new NodeTimeService()
+            {
+                private final ToLongFunction<TimeUnit> unixWrapper = NodeTimeService.unixWrapper(TimeUnit.NANOSECONDS, this::now);
+
+                @Override
+                public Node.Id id()
+                {
+                    return nodeId;
+                }
+
+                @Override
+                public long epoch()
+                {
+                    return ClusterMetadata.current().epoch.getEpoch();
+                }
+
+                @Override
+                public long now()
+                {
+                    return globalExecutor.nanoTime();
+                }
+
+                @Override
+                public long unix(TimeUnit unit)
+                {
+                    return unixWrapper.applyAsLong(unit);
+                }
+
+                @Override
+                public Timestamp uniqueNow(Timestamp atLeast)
+                {
+                    var now = Timestamp.fromValues(epoch(), now(), nodeId);
+                    if (now.compareTo(atLeast) < 0)
+                        throw new UnsupportedOperationException();
+                    return now;
+                }
+            };
+
+            this.store = new AccordCommandStore(0,
+                                                timeService,
+                                                new TestAgent.RethrowAgent(),
```

since we catch all exceptions in executors, Rethrowing is fine, don't need `errors` to propagate to the agent here